### ISSUE TITLE
Back button behavior in File manager

### ISSUE THREAD

**Issue by bahusoid** [koreader/koreader#4705]
* KOReader version: 2019.02
* Device: Android 7

From opened book I go to menu -> File Browser.
And from File browser I expect that Back button (hardware phone button) should return me back to my book. But it closes the app instead. Can it be mapped to "Open last document" action?

**Comment by Frenzie** [comment]
Note that you can customize the behavior in Settings (gear) → Navigation.

**Comment by bahusoid** [comment]
It would be cool if this dialog were integrated to "back history"

**Comment by Frenzie** [comment]
How do you mean exactly?

**Comment by Frenzie** [comment]
Note that from an Android UX paradigm randomly switching between "up" and "back" is correct behavior. (Yes, Android is stupid.)

The file manager has no back history at present, and note that there's no such thing as back history for a book you closed in any case.

**Comment by bahusoid** [comment]
I meant with "back history" enabled:
Open book 1
then open file manager
Open book 2

Press back button -> open file manager
Press back button -> open book 1

**Comment by Frenzie** [comment]
@pazos It'd be Android and AppImage, but since #4680 for all. ;-)

**Comment by Frenzie** [comment]
@bahusoid Yes, that's why I added a milestone to it. Technically speaking there'll be no back history involved though.

**Comment by pazos** [comment]
See https://www.mobileread.com/forums/showpost.php?p=3957792&postcount=6.
